### ISSUE TITLE
How can I apply the regularization for the learner

### ISSUE THREAD

**Issue by min6434** [Microsoft/CNTK#1609]
I am currently working with the 'CNTK 101: Logistic Regression and ML Primer'

At the training configuration, I can only set the 'learning rate' and there are no regularization parameters to add.

# Instantiate the trainer object to drive the model training
learning_rate = 0.5
lr_schedule = learning_rate_schedule(learning_rate, UnitType.minibatch) 
learner = sgd(z.parameters, lr_schedule)
trainer = Trainer(z, (loss, eval_error), [learner])

How can I apply the regularization at the learner?

**Comment by bencherian** [comment]
When setting up the learner you can supply keyword parameters for L1 and L2 regularization losses. For example, you could replace `sgd(z.parameters, lr_schedule)` with `sgd(z.parameters, lr_schedule, l2_regularization_weight=0.001)`.

These features are very clearly documented in the [learner documentation](https://www.cntk.ai/pythondocs/cntk.learners.html) for the Python API.

**Comment by min6434** [comment]
Thank you very much.
I will use that :)

**Comment by bencherian** [comment]
If the models have shared parameters, you can't jointly train then without specifying how to handle tradeoffs between optimizing for the different losses. If you just want to treat them all the same, just add the losses together.
________________________________

**Comment by min6434** [comment]
I dont know exactly what the shared parameter means but what I am doing is a simple logistic regression so I think just adding would do fine.
Thank you very much for your help.
